public/app/features/plugins/sandbox/constants.ts

```diff
@@ -0,0 +1 @@
+export const forbiddenElements = ['script', 'iframe'];
```

Comment:
Further testing is needed for plugins that could be injecting vendor libraries directly. I still could not find a valid case for a plugin using iframes.